packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -724,30 +720,195 @@ class TimeInput extends React.Component {
+    /**
+     * Get the mask string to place under the time input fields.
+     * @returns {String} a visual hint of the time format, like 'hh:mm'.
+     */
+    function mask() {
+      if (showSeconds) {
+        return intl.formatMessage({
+          id: 'Terra.timeInput.maskHourMinute',
+          defaultMessage: '(hh:mm:ss)',
+          description: 'A visual hint that the Time Input expects two-digit hour, minute and second.',
+        });
+      }
+      return intl.formatMessage({
+        id: 'Terra.timeInput.maskHourMinuteSecond',
+        defaultMessage: '(hh:mm)',
+        description: 'Like Terra.timeInput.maskHourMinute but no second.',
+      });
+    }
+
+    /**
+     * Get the description for the hour field.
+     *
+     * NOTE: the description does not take the place of validation or error messages.
+     * @returns {String} a description of the hour field, intended to be read to screen reader users.
+     */
+    function hourDescription() {
+      if (is12Hour) {
+        return intl.formatMessage({
+          id: 'Terra.timeInput.hourDescriptionTwelve',
+          defaultMessage: 'A two-digit 12-hour value',
+          description: `Explains to screen reader users that the hour field needs a two digit hour. This will be read
+          only when a screen reader is enabled. It is never displayed. It will be read when the user has focused on
+          the hour input or when the screen reader is reading the page to the user.`,
+        });
+      }
+      return intl.formatMessage({
+        id: 'Terra.timeInput.hourDescriptionTwentyFour',
+        defaultMessage: 'A two-digit 24-hour value',
+        description: 'Like Terra.timeInput.hourDescriptionTwelve but for a 24-hour clock.',
+      });
+    }
+
+    // Fan out some component-level props into input-level a11y attributes. See the big comment below for more info.
+    inputAttributes.isInvalid = isInvalid;
+    inputAttributes.disabled = disabled;
+    inputAttributes.required = required;
+
+    const a11yString = TimeUtil.getA11YTimeValue(this.props, this.state, this.postMeridiem);
+
+    function groupLabel() {
+      if (label) {
+        return intl.formatMessage({ // fix
+          id: 'Terra.timeInput.inputGroupValue',
+          defaultMessage: '{label}',
+          description: `This will be read by screen readers as the reader moves into the group of inputs. It is intended to
+      help the user understand "you are about to enter a section of the page where many different inputs all work
+      together for this one concept of time.`,
+        }, { label });
+      }
+      return intl.formatMessage({
+        id: 'Terra.timeInput.inputGroupValueDefault',
+        defaultMessage: 'Time',
+        description: `Same meaning as groupLabel, only for situations where the consumer has not provided a label for us
+      to plug in.`,
+      });
+    }
+
+    function hoursLabel() {
+      if(label){
```

Comment:
I would have thought this would have caused a lint error.
```suggestion
      if (label) {
```

Reply:
You're right. I'll double check my linter settings and try again.